lib/client/js/ui/Content.js

```diff
@@ -10,11 +10,11 @@ var Content = (function () {
      * @alias Menu
      */
     function Content() {
-        /** @member {Object} */
-        /** @member {Object} */
-        /** @member {Object} */
+        /** Object - @member {Header} */
```

Comment:
Sorry, dude, my last comment on this stuff was unclear. I used a hyphen to separate the doc snippet from the rest of the sentence. I meant that `@member {Header}` would suffice.